src/tools/pbs_output.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
```

Comment:
Not sure whether this should be part of PBSPro core package delivery. Personally i feel,  it shouldn't be. Explore the alternative design proposals

Reply:
This file is a RUR output plugin. Output of this file is used to update resource_used.energy value.

Reply:
Okays. Not sure about this. 

BTW .. did u tried creating RPM  and install ? do u need to update pbs.spec file.

Reply:
Fixed it.